drivers/storage/portworx/component/csi.go

```diff
@@ -598,6 +598,7 @@ func getCSIDeploymentSpec(
 			"--leader-election=true",
 			"--default-fstype=ext4",
 			"--extra-create-metadata=true",
+			"--timeout=5m",
```

Comment:
Since we maintain two different CSI provisioner sidecars, can you double check if this option will work for the older csi-provisioner image? This is to ensure, if a customer on PX 2.12 upgrades their operator to 23.5.1

Reply:
Yeah, this timeout flag has been there since provisioner 1.1.0:
https://github.com/kubernetes-csi/external-provisioner/blame/v1.1.0/cmd/csi-provisioner/csi-provisioner.go#L60